### PR TITLE
Add helm chart

### PR DESCRIPTION
**Pull request by FacuDeLorenzo**
Added helm chart to simplify deploy with kubernetes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,3 +118,80 @@ This project is licensed under either of
 at your option.
 
 
+# Helm deploy on EKS
+# Kubernetes on EKS
+## Create cluster
+```
+eksctl create cluster `
```
Comment (matzapata) [inline README.md:124]: Totally not a change request, but a shame not to do it in gcp to make it match the issuer deployment... Anyways trivial changes so it's okay

File: README.md [hunk 2]
```diff
@@ -118,3 +118,80 @@ This project is licensed under either of
...
+# Helm on windows
+## Configure env vars for helm
+```
+Set-Variable -Name APP_INSTANCE_NAME -Value rhs
+Set-Variable -Name NAMESPACE -Value default
+```
+
+## Install helm
+```
+helm install "$APP_INSTANCE_NAME" ./chart `
```
Comment (matzapata) [inline README.md]: I think this is actually installing the chart, can we rename to `install helm chart` and then split 2 sections for windows and linux based. Should be `export APP_INSTANCE_NAME=rhs`...

File: chart/values.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+rhs:
+  image:
+    repository: 447655722442.dkr.ecr.us-east-1.amazonaws.com/reverse-hash-service
```
Comment (matzapata) [inline chart/values.yaml]: can't we use a public docker image?

File: chart/values.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    tag: latest
+  service:
+    port: 8080
+
+db:
+  image:
+    repository: postgres
+    tag: latest
```
Comment (matzapata) [inline chart/values.yaml]: maybe better to fix some tag?

File: .gitignore
Comment (matzapata) [inline .gitignore:1]: needed?

**Review by matzapata — COMMENTED** [review]
Some comments but looks good!

File: .gitignore
Comment (FacuDeLorenzo) [reply .gitignore:1]: Just in case helm charts get installed, such as traefik on this case

File: chart/values.yaml (hunk 3 above)
Comment (FacuDeLorenzo) [reply chart/values.yaml]: Pushed to docker hub!

File: chart/values.yaml (hunk 4 above)
Comment (FacuDeLorenzo) [reply chart/values.yaml]: Fixed to 17-alpine version!

File: .gitignore
Comment (FacuDeLorenzo) [reply .gitignore:1]: Actually updated it so it looks like when we ignore node_modules:
`chart/charts/`

File: README.md (hunk 2 above)
Comment (FacuDeLorenzo) [reply README.md]: Splitted!

**Pull request merged**